egret/viz/generate_graphs.py

```diff
@@ -352,22 +354,21 @@ def _plot_generation_stack_components():
     ax.step(demand_indices, demand_by_hour, linewidth=3, color='#000000', where='post')
     
     # Add reserve requirements, if applicable.
-    reserve_requirements_by_hour = egret_model_data.data['system'].get('reserve_requirement')
+    reserve_requirements_array = attribute_to_array(egret_model_data.data['system'].get('reserve_requirement'))
     if reserve_requirements_by_hour is not None:
```

Comment:
I think this needs to be changed to `reserve_requirements_array` to fix the NameError